Bottom: 316f7376444b091b8a191964ec799e4bee52f2af
Top:    1e4bfb2ffabdd4abcd5bc64828ef9e9d85725b5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 07:26:18 +0100

Refresh of silly-space-navigator-interfac

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index c372549..f8d8a8a 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -741,8 +741,7 @@ void zoom_by (double factor, int x, int y); /* XXX: TEMPORARY HACK */
 void ndof_pan_cb (int dx, int dy, int dz, gpointer data)
 {
   if (dx || dy)
-    ghid_port_ranges_pan (-gport->zoom * 5 * dx,
-                          -gport->zoom * 5 * dy, TRUE);
+    ghid_pan_view_rel (-gport->zoom * 5 * dx, -gport->zoom * 5 * dy);
   if (dz)
     ghid_zoom_view_rel (gport->pcb_x, gport->pcb_y, 1.0 - (dz / 100.0));
 }
